packages/hooks/src/morpho/useMorphoVaultSupplierAddresses.ts

```
@@ -0,0 +1,129 @@
+async function fetchAllMorphoVaultSupplierAddresses(chainId: number): Promise<string[]> {
+  const allAddresses = new Set<string>();
+
+  // Fetch positions from all configured Morpho vaults
+  await Promise.all(
+    MORPHO_VAULTS.map(async vault => {
+      const vaultAddress = vault.vaultAddress[chainId];
+      if (!vaultAddress) return;
+
+      try {
+        const addresses = await fetchMorphoVaultSupplierAddresses(vaultAddress, chainId);
+        addresses.forEach(addr => allAddresses.add(addr));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)